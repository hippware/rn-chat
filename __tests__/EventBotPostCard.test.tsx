import 'react-native'
// import React from 'react'
// import renderer from 'react-test-renderer'
// import EventCard from '../src/components/event-cards/EventBotCard'
// import {Provider} from 'mobx-react/native'
// import {EventBotPost, Bot, BotPost} from 'wocky-client'
// import wocky from './utils/mockWocky'
// import mockLocationStore from './utils/mockLocationStore'

describe('EventBotPostCard', () => {
  it('renders', async () => {
    // await wocky.login('user', 'password', 'host')
    // const owner = wocky.profiles.get('user', {handle: 'test', avatar: null})
    // const bot = Bot.create({
    //   id: '1234',
    //   isSubscribed: false,
    //   title: 'title',
    //   server: 'server',
    //   radius: 30,
    //   owner,
    //   image: FileRef,
    //   description: 'description',
    //   location: {latitude: 1, longitude: 1},
    //   address: 'address',
    //   followersSize: 0,
    //   totalItems: 0,
    //   // addressData: types.optional(Address, {}),
    //   // subscribers: types.optional(ProfilePaginableList, {}),
    //   // posts: types.optional(BotPostPaginableList, {}),
    //   error: '',
    // })
    // mockLocationStore.setPosition({longitude: 2, latitude: 2, accuracy: 2})
    // const post = BotPost.create({
    //   id: '123',
    //   time: 123,
    //   loaded: true,
    //   content: 'some content',
    //   title: 'a title',
    //   profile: owner,
    // })
    // const botPost = EventBotPost.create({id: '123', cursor: '123', time: 123, bot, post})
    // const toRender = (
    //   <Provider locationStore={mockLocationStore}>
    //     <EventCard item={botPost} />
    //   </Provider>
    // )
    // const tree = renderer.create(toRender).toJSON()
    // expect(tree).toMatchSnapshot()
  })
})
