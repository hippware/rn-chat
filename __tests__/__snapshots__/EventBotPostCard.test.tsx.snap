// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventBotPostCard renders 1`] = `
<View
  accessible={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "flexDirection": "row",
      "marginTop": 20,
      "paddingHorizontal": 15,
    }
  }
>
  <View
    style={
      Object {
        "marginLeft": 10,
        "marginRight": 20,
        "paddingRight": 26,
      }
    }
  >
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "justifyContent": "flex-end",
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Array [
            undefined,
            Object {
              "height": 84.60000000000001,
              "width": 84.60000000000001,
            },
          ]
        }
      >
        <BVLinearGradient
          colors={
            Array [
              "rgb(242,68,191)",
              "rgb(254,110,98)",
              "rgb(254,92,108)",
            ]
          }
          end={
            Object {
              "x": 1,
              "y": 0,
            }
          }
          start={
            Object {
              "x": 0,
              "y": 1,
            }
          }
          style={
            Object {
              "borderRadius": 42.300000000000004,
            }
          }
        >
          <View
            style={
              Array [
                Object {
                  "borderColor": "rgb(255,255,255)",
                  "borderRadius": 42.300000000000004,
                  "borderWidth": 3.6,
                  "height": 84.60000000000001,
                  "overflow": "hidden",
                  "width": 84.60000000000001,
                },
                Object {
                  "alignItems": "center",
                  "justifyContent": "center",
                },
              ]
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "color": "rgb(255,255,255)",
                  },
                  Object {
                    "fontFamily": "Roboto-Regular",
                    "fontSize": 20,
                  },
                ]
              }
            >
               
            </Text>
          </View>
        </BVLinearGradient>
      </View>
    </View>
    <Image
      source={
        Object {
          "testUri": "../../../images/notificationMessage.png",
        }
      }
      style={
        Object {
          "bottom": 10,
          "height": 46.800000000000004,
          "position": "absolute",
          "right": 0,
          "width": 46.800000000000004,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "flex-start",
        "alignSelf": "center",
        "flex": 1,
        "marginRight": 14,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": undefined,
            "fontFamily": "Roboto-Regular",
            "fontSize": 12,
          },
          undefined,
        ]
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": undefined,
              "fontFamily": "Roboto-Medium",
              "fontSize": 13,
            },
            undefined,
          ]
        }
      >
        @null 
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "rgb(114,100,109)",
              "fontFamily": "Roboto-Regular",
              "fontSize": 13,
            },
            undefined,
          ]
        }
      >
        commented on
         
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "rgb(155,155,155)",
              "fontFamily": "Roboto-Light",
              "fontSize": 13,
            },
            undefined,
          ]
        }
      >
        1234
      </Text>
    </Text>
  </View>
  <View
    style={
      Object {
        "height": 40,
        "justifyContent": "center",
        "width": 71,
      }
    }
  />
</View>
`;
